irods_testing_environment/irods_setup.py

```diff
@@ -413,11 +437,16 @@ def restart_rsyslog(container):
 
 
 def stop_irods(container):
+    # TODO Need alternative to irodsctl.
     irodsctl = os.path.join(context.irods_home(), 'irodsctl')
     return execute.execute_command(container, f'{irodsctl} stop', user='irods')
+    # TODO This is NOT equivalent to irodsctl because it doesn't block.
+    # We can either keep irodsctl for cases like this or we have to rethink our approach or build new tools.
+    #return execute.execute_command(container, f'kill -TERM $(cat /var/run/irods-server.pid)', user='irods')
 
 
 def restart_irods(container):
+    # TODO Need alternative to irodsctl.
     irodsctl = os.path.join(context.irods_home(), 'irodsctl')
```

Comment:
Need to think about this.

As discussed with @trel, calling `IrodsController()` via `python3 -c` covers both versions of iRODS and removes the need for `irodsctl`.

Reply:
https://github.com/irods/irods/issues/7996

Reply:
Replaced irodsctl with IrodsController.